mmv1/third_party/terraform/tests/resource_workstations_workstation_config_test.go.erb

```diff
@@ -0,0 +1,245 @@
+<% autogen_exception -%>
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
```

Comment:
A solution to the erroring check on this PR would be to change this file to have a single set of `<% unless version == "ga" -%>` and `<% end -%>` that contains all the code _except_ `package google`. In the GA provider the file would only contain `package google` and in the beta provider the full file would be present.

Reply:
Hey @SarahFrench, thanks for the suggestion. I'll apply it straight away. Hope you had a nice time off, welcome back :).

Reply:
I've just moved the `<% unless version == "ga" -%>` back-up above the imports, I suppose this would be sufficient too? If so, could you please trigger the /gcbrun :)

Reply:
Will do!